subsys/bluetooth/controller/ll_sw/ull_adv.c

```diff
@@ -1509,8 +1511,11 @@ uint8_t ll_adv_enable(uint8_t enable)
 						EVENT_OVERHEAD_START_US +
 						(EVENT_TICKER_RES_MARGIN_US << 1));
 
+				ticks_slot_overhead =
+					ull_adv_sync_evt_init(adv, sync, NULL);
```

Comment:
```suggestion
				ticks_slot_overhead = ull_adv_sync_evt_init(adv, sync, NULL);
```

The TSC decided this week to remove the 80 character suggestion, so now we can (and should) use the 100 character limit instead.